scripts/build-appbuilder.sh

```diff
@@ -31,55 +112,74 @@ if [ ! -z "$deploy" ]; then
     fi
 fi
 
-# Check if sentryconfig.local.ts exists, copy it to sentryconfig.ts if it does
-build_timestamp=$(date +'%Y-%m-%d_%H:%M')
-if [ -f "sentryconfig.local.ts" -a ! -z "${deploy}" ]; then
-    echo "Configuring sentry for build timestamp: ${build_timestamp}"
-    mv sentryconfig.ts sentryconfig.ts.bak
-    sed -e "s/BUILD_TIMESTAMP/${build_timestamp}/" sentryconfig.local.ts > sentryconfig.ts
+# If we are deploying, check for git changes
+if [ ! -z "$deploy" ]; then
+    if [[ -n $(git status --porcelain) ]]; then
+        echo "There are uncommitted changes."
+        exit 1
+    fi
 fi
 
-prefix=v1/main
+# Get the current branch
+branch=$(git rev-parse --abbrev-ref HEAD)
 
-echo "Building AppBuilder version $version with prefix $prefix"
-vite build --base=$prefix/$version/
-if [ $? -ne 0 ]; then
-    echo "Build failed."
-    exit 1
-fi
+# npm version
+npm_version=$(node -p "require('./package.json').version")
+echo "Current npm version: $npm_version"
 
-if [ -z "${deploy}" ]; then
-    echo "Skipping deployment."
-    exit 0
-fi
+deploying_branch=1
 
-if [ -z "$APPBUILDER_BUCKET" ]; then
-    echo "APPBUILDER_BUCKET environment variable is not set."
-    exit 1
-fi
+# If the branch is "development", "staging" or starts with "task/", we use the branch name as the version
+if [ "$branch" == "development" ] || [ "$branch" == "staging" ]; then
+    deploying_branch=1
+    version=$branch
+elif [[ $branch == task/* ]]; then
+    deploying_branch=1
+    # In this case we have to remove the "task/" prefix
+    version=${branch#task/}
+elif [[ $branch == "latest" ]]; then
```

Comment:
@MajorMeerkatThe3rd let's call the branch `master`, which we use everywhere else, but deploy to `latest` in that case.